### ISSUE TITLE
Can't build

### ISSUE THREAD

**Issue by clarkezone** [Aminator9000/DirectX12GameEngine#6]
I cloned the repo, I have all the dependencies installed per the readme, the GLTF loader project won't compile:

```1>------ Build started: Project: GltfLoader, Configuration: Debug Any CPU ------
1>Interface.cs(8,18,8,24): error CS0234: The type or namespace name 'Schema' does not exist in the namespace 'GltfLoader' (are you missing an assembly reference?)
1>Interface.cs(30,33,30,37): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(43,33,43,37): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(166,52,166,56): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(201,52,201,56): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(178,49,178,53): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(251,49,251,53): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(217,66,217,72): error CS0246: The type or namespace name 'Schema' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(226,57,226,63): error CS0246: The type or namespace name 'Schema' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(271,49,271,54): error CS0246: The type or namespace name 'Image' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(287,33,287,37): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(297,50,297,54): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(307,43,307,47): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(318,43,318,47): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(332,49,332,53): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(344,49,344,53): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(356,49,356,53): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(416,55,416,59): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Interface.cs(430,55,430,59): error CS0246: The type or namespace name 'Gltf' could not be found (are you missing a using directive or an assembly reference?)
1>Done building project "GltfLoader.csproj" -- FAILED.```

**Comment by Aminator** [comment]
This is happening because you need to run the generator first that creates C# classes from the GLTF schema. I will fix this in the next commit by putting that generation step in an MSBuild task.

**Comment by Aminator** [comment]
This issue should now be fixed in commit 905f88990a23320b9d673a62cec352671dc99ecd.
